### ISSUE TITLE
NPE using new Jsr310Module()

### ISSUE THREAD

**Issue by gadton** [modelmapper/modelmapper-module-java8#3]
Registering an instance of Jsr310Module using the default ctor results in a NPE during mapping from `String` to `LocalDateTime` as Jsr310ModuleConfig.dateTimePattern is null.

```
final FooDTO fooDto = new FooDTO();
fooDto.setBar( "2018-10-10 05:05:05" );
new ModelMapper().registerModule( new Jsr310Module() ).map( fooDto, Foo.class );
```
Passing a Jsr310ModuleConfig in the ctor explicitly using `Jsr310ModuleConfig.builder().build()` will solve the issue.

**Comment by ramasCoder** [comment]
This involves changing the default constructor for Jsr310Module class as follows if I am not wrong  :
public Jsr310Module() {
    this(Jsr310ModuleConfig.builder().build());
}
Right ?

**Comment by chhsiao90** [comment]
The issue was fixed in v1.2.0.
